@xen-orchestra/web-core/lib/components/ui/input/UiInput.vue

```diff
@@ -1,141 +1,179 @@
-<!-- WIP -->
+<!-- v5 -->
 <template>
-  <div class="ui-input">
-    <VtsIcon :icon accent="current" class="before" />
-    <input :id v-model.trim="modelValue" class="typo p1-regular input" type="search" v-bind="attrs" />
-    <VtsIcon
-      v-if="!attrs.disabled && modelValue"
-      :icon="faXmark"
-      class="after"
-      accent="info"
-      @click="modelValue = ''"
-    />
+  <div class="ui-input" :class="toVariants({ accent })">
+    <UiLabel v-if="slots.default || label" :accent="labelAccent" :required :icon="labelIcon" :href :html-for="inputId">
```

Comment:
When it will be available, the `UiLabel` would expose the `:for` prop:
```suggestion
    <UiLabel v-if="slots.default || label" :accent="labelAccent" :required :icon="labelIcon" :href :for="id">
```